### PR TITLE
CLI Updates for Preview 7 - Part 1

### LINKED ISSUE #8860: pack argument accepts project or solution
[This stackoverflow question has a comment](https://stackoverflow.com/a/52935200/2843065) from a customer who was confused by the docs because it claims the argument only accepts a project file, but testing with a solution file, it packs all the projects in the solution (at least in my simple 2 classlib project solution). I think it would be good if the docs (and `dotnet pack -h`) were updated to mention that the argument could also be a sln file.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: a7e805a3-f2ec-b05a-d5c8-29f7a8c16a5f
* Version Independent ID: 4257ec12-df6f-ec52-aad3-012e5fce8133
* Content: [dotnet pack command - .NET Core CLI](https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-pack?tabs=netcore2x)
* Content Source: [docs/core/tools/dotnet-pack.md](https://github.com/dotnet/docs/blob/master/docs/core/tools/dotnet-pack.md)
* Product: **dotnet-core**
* GitHub Login: @mairaw
* Microsoft Alias: **mairaw**

### LINKED ISSUE #10265: "--include-source" implies "--include-symbols"
This documentation should mention that --include-symbols implies --include-symbols, and that the "src" folder is only added to the symbols package.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: a7e805a3-f2ec-b05a-d5c8-29f7a8c16a5f
* Version Independent ID: 4257ec12-df6f-ec52-aad3-012e5fce8133
* Content: [dotnet pack command - .NET Core CLI](https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-pack?tabs=netcore2x)
* Content Source: [docs/core/tools/dotnet-pack.md](https://github.com/dotnet/docs/blob/master/docs/core/tools/dotnet-pack.md)
* Product: **dotnet-core**
* GitHub Login: @mairaw
* Microsoft Alias: **mairaw**

### LINKED ISSUE #14163: nologo inconsistency
The synopsis says the switch is `--nologo`.

The arguments say it's only available since .NET Core 3.0, but `-nologo` (note single dash) works using .NET Core 2.2.6.

The arguments say it's `--no-logo` (note extra dash), but that doesn't work in any version (tested on 2.2.6 and 3.0.0 preview 8).

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 910cd365-d63a-00dc-417c-db017f83bf6f
* Version Independent ID: 5e495ddd-140e-de5c-a793-d4a9f88be51e
* Content: [dotnet build command - .NET Core CLI](https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-build)
* Content Source: [docs/core/tools/dotnet-build.md](https://github.com/dotnet/docs/blob/master/docs/core/tools/dotnet-build.md)
* Product: **dotnet-core**
* Technology: **dotnet-cli**
* GitHub Login: @mairaw
* Microsoft Alias: **mairaw**

### PR DESCRIPTION
**Pull request by mairaw**
Fixes #8860 
Fixes #10265
Fixes #14163 
Contributes to #13281 and #9893

### CODE REVIEW

File: docs/core/tools/dotnet-pack.md [hunk 1]
```diff
@@ -53,91 +51,59 @@ Web projects aren't packable by default. To override the default behavior, add t
 
 ## Arguments
```
Comment (KathleenDollard) [inline docs/core/tools/dotnet-pack.md]: @livarcocc Can you or someone on your team review this section for the versions that things first appeared in, if it wasn't 1. Some are missing or look wrong to me.

Comment (mairaw) [reply docs/core/tools/dotnet-pack.md]: Oh I've missed some of the x.x that I left as placeholders. But yes, reviewing for accuracy would be nice.

Comment (mairaw) [reply docs/core/tools/dotnet-pack.md]: I've fixed the missing version and also double checked the changes I've made for the 2.0 SDK release [here](https://github.com/dotnet/docs/commit/81b08d3c9acf0b3534047910aa8ff9bf8559940d#diff-8071d44406b4c597993915f7deca9fbb). It seems those options were introduced for 2.0.

I can't download the 2.0 SDK installer any longer and I don't have it on my machine.

File: docs/core/tools/dotnet-build.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ---
 title: dotnet build command
 description: The dotnet build command builds a project and all of its dependencies.
-ms.date: 04/24/2019
+ms.date: 08/08/2019
```
Comment (BillWagner) [inline docs/core/tools/dotnet-build.md]: You should update this before merging.

File: docs/core/tools/dotnet-help.md [hunk 3]
```diff
@@ -1,11 +1,15 @@
 ---
 title: dotnet help command
 description: The dotnet help command shows more detailed documentation online for the specified command.
-ms.date: 12/04/2018
+ms.date: 08/08/2019
```
Comment (BillWagner) [inline docs/core/tools/dotnet-help.md]: Also update before merging.

File: docs/core/tools/dotnet-migrate.md [hunk 4]
```diff
@@ -1,23 +1,16 @@
 ---
 title: dotnet migrate command
 description: The dotnet migrate command migrates a project and all of its dependencies.
-ms.date: 06/26/2019
+ms.date: 08/08/2019
```
Comment (BillWagner) [inline docs/core/tools/dotnet-migrate.md]: I'll stop mentioning, but all ms.date fields should be updated before merging.

**Review by BillWagner — APPROVED** [review]
This LGTM @mairaw 

You can :shipit: when ready.

File: docs/core/tools/dotnet-migrate.md (hunk 4 above)
Comment (mairaw) [reply docs/core/tools/dotnet-migrate.md]: LOL thanks! I'll fix those.

Comment (mairaw) [reply docs/core/tools/dotnet-migrate.md]: Oh but I did change the ms.date. I"ll double check for Preview 9/RTM if anything changed again and do a pass across all CLI ref. This date is good as-is.

**Comment by mairaw** [comment]
@KathleenDollard @dsplaisted do you want to double check these CLI changes before I merge?

**Review by KathleenDollard — COMMENTED** [review]
@dsplaisted it would be great if you could also look this over. 

Did we use --nologo? Posix would be --no-logo, but whatever we did we did and should just document. 

LGTM

**Comment by mairaw** [comment]
That's what I see at least:
image

**Comment by KathleenDollard** [comment]
I assume it went in as --nologo, and I filed an issue about why it is so ugly in help. 

@livarcocc

**Comment by imba-tjd** [comment]
Fix https://github.com/dotnet/docs/issues/14163

**Comment by mairaw** [comment]
Good catch @imba-tjd!

**Review by dsplaisted — APPROVED** [review]

**Pull request merged**